### PR TITLE
Cluster ddl performance

### PR DESCRIPTION
**Pull request by dothebart**
will create a json from the [cluster ddl csv](https://github.com/arangodb/simple-performance-test/blob/master/ddl/cluster_database_create.js)
```json
{
  "test": {
    "lastRun": {
      "average": 11.243625688552857,
      "median": 10.64977216720581,
      "min": 6.550904750823975,
      "max": 16.084526777267456,
      "deviation": {
        "pst": 2.7485627305518263,
        "pvariance": 7.554597083778512,
        "stdev": 2.8199658608942344,
        "variance": 7.952207456608961
      },
      "values": [
        6.550904750823975,
        8.282733678817749,
        9.04445767402649,
        8.355422019958496,
        8.65219783782959,
        8.431092739105225,
        9.286822319030762,
        9.16083025932312,
        9.8131263256073,
        9.970505952835083,
        11.329038381576538,
        12.30403733253479,
        12.769443273544312,
        14.026540040969849,
        13.154082775115967,
        13.322911262512207,
        14.180629968643188,
        15.224415302276611,
        16.084526777267456,
        14.928795099258423
      ],
      "numberRuns": 20
    },
    "dblast": {
      "average": 0.8857569575309754,
      "median": 0.8639727830886841,
      "min": 0.43496227264404297,
      "max": 1.3073265552520752,
      "deviation": {
        "pst": 0.24384318060678578,
        "pvariance": 0.05945949672843355,
        "stdev": 0.2501778246061527,
        "variance": 0.06258894392466689
      },
      "values": [
        0.43496227264404297,...
      ],
      "numberRuns": 20
    },
    "dbAvg": {
      "average": 0.6926441245659662,
      "median": 0.6903014399788596,
      "min": 0.43496227264404297,
      "max": 0.8857569575309754,
      "deviation": {
        "pst": 0.11870293673736847,
        "pvariance": 0.014090387190075701,
        "stdev": 0.1217866434214736,
        "variance": 0.01483198651586916
      },
      "values": [
        0.43496227264404297,...
      ],
      "numberRuns": 20
    },
    "colLast": {
      "average": 4.453278648853302,
      "median": 4.431670904159546,
      "min": 2.8084986209869385,
      "max": 6.249857425689697,
      "deviation": {
        "pst": 0.9318175260398781,
        "pvariance": 0.8682839018350789,
        "stdev": 0.9560246098104593,
        "variance": 0.9139830545632409
      },
      "values": [
        2.8084986209869385,...
      ],
      "numberRuns": 20
    },
    "colAvg": {
      "average": 3.7260375417996245,
      "median": 3.6805115818977354,
      "min": 2.8084986209869385,
      "max": 4.453278648853302,
      "deviation": {
        "pst": 0.43519381423614223,
        "pvariance": 0.18939365594940188,
        "stdev": 0.4464994323676499,
        "variance": 0.19936174310463356
      },
      "values": [
        2.8084986209869385,...
      ],
      "numberRuns": 20
    },
    "idxLast": {
      "average": 5.8925384163856505,
      "median": 5.357584714889526,
      "min": 3.3002872467041016,
      "max": 8.509495496749878,
      "deviation": {
        "pst": 1.59625846875925,
        "pvariance": 2.5480410990856255,
        "stdev": 1.6377266332795868,
        "variance": 2.68214852535329
      },
      "values": [
        3.3002872467041016,...
      ],
      "numberRuns": 20
    },
    "idxAvg": {
      "average": 4.680370885734498,
      "median": 4.491986429691314,
      "min": 3.3002872467041016,
      "max": 5.8925384163856505,
      "deviation": {
        "pst": 0.6908845224544478,
        "pvariance": 0.47732142336711025,
        "stdev": 0.7088325638289529,
        "variance": 0.5024436035443266
      },
      "values": [
        3.3002872467041016,...
      ],
      "numberRuns": 20
    }
  },
  "size": {
    "collection": "7.9792211055755615",
    "count": 20,
    "size": 0
  },
  "configuration": {
    "version": "devel",
    "branch": "3.8",
    "mode": "cluster",
    "edition": "enterprise"
  },
  "isoDate": "2020-07-27T11:52:33",
  "date": 1595843553,
  "ms": 1595843553000
}
```

### CODE REVIEW

File: jenkins/helper/performance2json.py [hunk 1]
```diff
@@ -159,12 +163,75 @@ def simple_performance_cluster(row):
...
+                        values[header] = []
+        elif rownum == -1:
+                count = len(values[headline[0]])
+                number_runs = len(values[headline[0]])
+                date = current_date
+                size = 0
+
+                result = {
+                        "test": {
+                        },
```
Comment (fceller) [inline jenkins/helper/performance2json.py:181]: You can add a name here: for example

"test": { "name": "ddl" }

File: jenkins/helper/performance2json.py [hunk 2]
```diff
@@ -159,12 +163,75 @@ def simple_performance_cluster(row):
...
+                        "size": {
+                                "collection": row[6],
+                                "count": count,
+                                "size": size
+                        },
```
Comment (fceller) [inline jenkins/helper/performance2json.py:186]: what is the meaning of size in this test?

Comment (dothebart) [reply jenkins/helper/performance2json.py:186]: ok, this was still a question of mine to be asked - what should be put in there? the test parameters like collection count, index count, loop count?

Comment (fceller) [reply jenkins/helper/performance2json.py:186]: Yes, exactly.

**Pull request merged**